gradle/dependencies.gradle

```diff
@@ -194,7 +149,6 @@ ext.deps = [
         'gen'      : gen,
         'runtime'  : runtime,
         'test'     : test,
-        'versions' : versions,
```

Comment:
So everything except this line is backward-compatible. Can we please highlight in the PR description that this is a breaking change and developers should use versions from respective Kotlin objects?